pkgs/tools/networking/corerad/default.nix

```diff
@@ -2,18 +2,16 @@
 
 buildGo118Module rec {
```

Comment:
This should be ideally done in top-level.

Reply:
Thanks! Opened #167881.